### ISSUE TITLE
Inlining across modules with webpack?

### ISSUE THREAD

**Issue by zrait** [terser/terser#1079]
**Bug report or Feature request?** Feature request

I'm using `terser` via `terser-webpack-plugin` and it seems unable to perform inlining across modules once transformed into the webpack manifest and thereby prevents dead code elimination for a dev logging function I'm trying to compile out when `NODE_ENV !== 'development`

When I tried creating a minimal reproduction using the same config, I ended up with generated code that didn't include the webpack module loader infrastructure and the code was removed as expected.

**Version (complete output of `terser -V` or specific git commit)**
5.7.2

**Complete CLI command or `minify()` options used**

```js
compress: {
  passes: 3,
  module: true,
},
```

**`terser` input**
Logging.js
```js
export const devLog= process.env.NODE_ENV === 'development' ? console.log: () => {};
```

SomeOtherFile.js
```js
import { devLog } from './Logging';
...
devLog('hello');
```

**`terser` output or error**
bundle.js
```js
...
52244: (__unused_webpack_module, __webpack_exports__, __webpack_require__) => {
        __webpack_require__.d(__webpack_exports__, {
...
(0, Logging.Cv)('hello').
...
70209: (__unused_webpack_module, __webpack_exports__, __webpack_require__) => {
        __webpack_require__.d(__webpack_exports__, { Cv: () => devLog });
        var devLog = function () {};
      },
```

**Expected result**
The string 'hello' should be eliminated from the output

**Comment by fabiosantoscode** [comment]
Hello there!

Webpack's output is really hard to follow, so Terser doesn't follow it.

There's quite a few issues that talk about this, so I'm closing.

Thanks for reaching out!
